src/watch_filter.rs

```diff
@@ -173,15 +200,54 @@ impl Filterer for WatchFilter {
         }
 
         // By process of elimination the event is valid
-        tracing::trace!("passing event: {event:?}");
+        tracing::debug!("passing event: {event:?}");
         Ok(true)
     }
 }
 
+#[derive(Debug)]
+pub struct Config {
```

Comment:
I am a bit surprised that the docs thing is not nagging about things being public without doc comments.  I'm not too fussed but it will certainly be needed when you crate it.

Reply:
The module isn't public, so the struct isn't crate-public.